pkg/catalog/ingress.go

```diff
@@ -124,6 +124,7 @@ func (mc *MeshCatalog) GetIngressTrafficPolicy(svc service.MeshService) (*traffi
 				HTTPRouteMatch:   trafficpolicy.WildCardRouteMatch,
 				WeightedClusters: mapset.NewSet(backendCluster),
 			},
+			// TODO(steeling): one place its set
```

Comment:
I don't understand this comment. Could you rephrase or remove this?
Also, it's preferable for TODO comments to reference an issue, e.g. `TODO(#<issue>): ...`

Reply:
that was a personal note that snuck in, removed!